example.json

```diff
@@ -7,14 +9,15 @@
     "_homeButton": {
         "_isEnabled": false,
         "_hideHomeButton": false,
-        "_redirectToId": "",
+        "_comment": "Amend co-00 to match the ID of the start / landing page",
+        "_redirectToId": "co-00",
         "alt": "Home"
     }
 
     // contentObjects.json
     "_homeButton": {
-        "_isEnabled": false,
+        "_isEnabled": true,
```

Comment:
Maybe change `hideBackButton` to `true` for this example?

Reply:
I would actually argue that `_hideHomeButton` should be set to `true` and `_hideBackButton` set to `false` to retain the standard Adapt back button functionality. Though that all depends upon the primary use case of the plugin which I'm assuming is to allow users to navigate back to a landing / start page from a menu.

Reply:
> which I'm assuming is to allow users to navigate back to a landing / start page from a menu.

@guywillis the configuration for that is shown in the previous example

Reply:
Amended example as per suggestion